test/setupTests.js

```diff
@@ -12,7 +12,7 @@ jest.mock('ora', () =>
 )
 jest.mock('conf')
 jest.mock('inquirer')
-jest.mock('execa')
+jest.mock('execa', () => jest.fn(() => Promise.resolve({ stdout: 'master' })))
```

Comment:
I would leave this as a general mock like this:

```suggestion
jest.mock('execa')
```

And then mock on the tests as needed the parts of the module you'll need to mock